README.md

```diff
@@ -46,6 +46,9 @@ USAGE: <XeroSDK INSTANCE>.<API_NAME>.<API_METHOD>(<PARAMETERS>)
 # Get a list of all Invoices
 response = connection.invoices.get_all()
 
+# Get a list of all Invoices using generator
+response = connection.invoives.list_all_generator()
```

Comment:
invoices* typo. Also this isn't a response, add an iterator and show a very simple example of this generator.